llvm/test/Transforms/LowerMatrixIntrinsics/reduce.ll

```diff
@@ -0,0 +1,200 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -passes='lower-matrix-intrinsics' -S < %s | FileCheck %s
+
+define i32 @reduce_add(ptr %in, ptr %out) {
```

Comment:
should also have at least one test where have have more than 2 columns and one with an odd number of columns